students/JimJenkins/session09/mailroom.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3
+
+"""
+Author: Jim Jenkins
+Date: 12/10/2018
+"""
+
+import cli_main
+
+if __name__ == '__main__':
```

Comment:
and yo don't want an if __name__ block here at all, either -- the top level program will be the cli_main file -- over even a file that does nothing but call that.